src/weather_model_graphs/create/mesh/mesh.py

```diff
@@ -139,9 +137,7 @@ def create_multirange_2d_mesh_graphs(
     G_all_levels = []
     for lev in range(mesh_levels_to_create):  # 0-index mesh levels
         # Compute number of nodes on level separate for each direction
-        nodes_x, nodes_y = (
-            nleaf / (grid_refinement_factor * (level_refinement_factor**lev))
-        ).astype(int)
+        nodes_x, nodes_y = (nleaf / (level_refinement_factor**lev)).astype(int)
```

Comment:
Hmm, not obvious to me that that is desirable. There's two possible behaviors in terms of how many levels can be created: We stop making new levels when one of the directions (x or y) can't be refined further, or we stop making new levels when none of the directions can be refined further. Not obvious which one is preferable to me, but I don't see myself wanting to create graphs following the second choice. One could make this an option. Or just go with the second option and then you have to use the max level argument to get rid of the top levels you don't want.

Anyway I think that a change to this would require changing the number of level computations above as well, for that to be correct. It should probably also be combined with separate level_refinement_factors for x- and y-directions.

My preference is to leave this as is in this PR, and can be tweaked later.

Reply:
Ok, good points. I agree. Thanks for typing this out. Let leave as it is for now and people can adjust if they want to